src/main/java/pro/sky/JavaCoredemo2/controller/JavaQuestionController.java

```diff
@@ -0,0 +1,36 @@
+package pro.sky.JavaCoredemo2.controller;
+
+import pro.sky.JavaCoredemo2.entity.Question;
+import pro.sky.JavaCoredemo2.services.JavaQuestionService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/exam/java")
+public class JavaQuestionController {
+
+    private final JavaQuestionService service;
```

Comment:
```suggestion
    private final QuestionService service;
```
инъекцию надо всегда делать по интерфейсу, а не реализации

Reply:
также см. внимательнее задание про Qualifier аннотацию